src/components/core/DpEditor/DpEditor.vue

```diff
@@ -577,6 +580,22 @@ export default {
       default: false
     },
 
+    /**
+     * Needed to get the Route, when any of these Props are enabled in the DpEditor
+     */
+    routes: {
+      type: Object,
+      required: false,
+      default: () => ({}),
+      validator: (prop) => {
+        return Object.keys(prop).every(key => [
+            'boilerplateListRoute',
+            'getFileByHash',
+            'similarRecommendationsRoute'
+        ].includes(key))
```

Comment:
i am not shure, but maybe this makes it even better understandable:
```suggestion
      validator: (prop) => {
        return typeof prop.boilerplateListRoute === 'string' 
          && typeof prop.getFileByHash === 'function'
          && typeof prop.similarRecommendationsRoute === 'string'
     }
```
but weather this nor the original validation covers that some routes are optional

Reply:
Revert Commit suggestion in https://github.com/demos-europe/demosplan-ui/pull/91/commits/db654ecf07ac1e708303ad7c9e37f5d912ddb016